storage/wavefront.go

```diff
@@ -41,9 +41,14 @@ func Wavefront(userInput map[string]string, wg *sync.WaitGroup) {
 
 	if proxyHost != "" {
 		Debug(userInput, fmt.Sprintf("Using proxy host: %s", proxyHost))
+		proxyMetricPort, err := strconv.Atoi(userInput["storage_driver_wf_metric_port"])
+		if err != nil {
+			fmt.Println("Setting metric port to 2878, as it is not supplied or supplied value is invalid")
+			proxyMetricPort = 2878
+		}
 		proxyCfg := &wavefrontSenders.ProxyConfiguration{
 			Host:                 proxyHost, // Proxy host IP or domain name
```

Comment:
This only partially addresses the issue raised in INT-701 (making the port configurable). 

The other issue was if a user by mistake sets the proxyHost to be `hostname:2878` we then pass it to the SDK as is and it resolves to `hostname:2878:2878`. Can you add some error checking around the proxyHost value supplied by the user? Throw an error if port is included in the hostname.